test/suites/basic.sh

```diff
@@ -826,8 +826,8 @@ test_service_mismatch() {
 
   # Install microceph and microovn on the first machine only.
   for m in micro02 micro03 ; do
-    lxc exec "${m}" -- snap remove microceph --purge
-    lxc exec "${m}" -- snap remove microovn --purge
+    lxc exec "${m}" -- snap disable microceph
```

Comment:
Can it cause some leftovers if we just disable/enable the snaps? 
Why not running ` snap install --dangerous "/root/microovn.snap"` here?

Reply:
It's overall quicker to simply disable/re-enable the snaps, which is what we do all over the test suite. This one spot was an anomaly.